skimage/novice/__init__.py

```diff
@@ -96,6 +96,12 @@
 >>> picture.compare()  # doctest: +SKIP
 
 """
+
+import warnings
+
+warnings.warn("This module was deprecated in version 0.14. "
+              "It will be removed in 0.16.")
+
 from ._novice import Picture, open, colors, color_dict
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports -> `Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.`

Reply:
OK, I have removed the spurious blank line.

Reply:
Sorry, I meant that `import warnings` and `from ._novice import Picture, open, colors, color_dict` should be put before the `.warn` call.

Reply:
Is it OK now?

Reply:
Perfect, thanks :)